tests/ashley/test_templatetags_custom_tags.py

```diff
@@ -160,36 +136,78 @@ def get_html_post_by(user):
         self._init_forum(forum, context)
 
         # Set up topic
-        topic1 = TopicFactory(forum=forum, poster=user1, subject="topic by instructor")
+        topic1 = TopicFactory(
+            forum=forum, poster=user1, subject="Je ne suis pas votre élève"
+        )
```

Comment:
Actually you don't need to worry about the value of the subject string since this is not the subject of this test...
```suggestion
        topic1 = TopicFactory(forum=forum)
```

Reply:
ok I kept the relation with the poster though !